sdk/storage/azure-storage-common/vcpkg/vcpkg.json

```diff
@@ -22,7 +22,8 @@
     },
     {
       "name": "openssl",
-      "platform": "!windows"
+      "platform": "!windows",
+      "version" : "1.1.1n" 
```

Comment:
Are storage and core the only components which use openssl? I'm not seeing Attestation in this PR, and I know it uses OpenSSL.

Have you updated the cgmanifest.json files for those components to include the hash of this new version of OpenSSL?

Reply:
i will let you update open sll for attestation, it is not specified in vcpkg dependencies, or any other place for that matter as a dep with a version. 
The request i got from @RickWinter was to update the vcpkg dependencies version, not make any code changes. 
If that is the case then i will close this PR and we have to properly schedule and prioritize this item as i am unfamiliar with the attestaion code.

Reply:
Also cg manifest seems to be orthogonal to vcpkg, those seem to be packages that are not specified in vcpkg and vice versa, Also attestetion while asking for openssl in the readme it does not specify it in the vcpkg.json. 
If it was intended to be there we can add it , although we would need to ensure the accompanying find_package is specified in the cmake files.

Reply:
cg manifest is slightly orthogonal. All packages which use any open source components MUST have their open source use documented in the cgmanifest.json. And that entry is triggered based on the commit hash that the package was based on. By updating the cgmanifest.json file, it forces the person who updated the package version (that would be you :)) to dig up the hash and document it.

I just went through this yesterday when I updated the cgmanifest.json for attestation, it's not hard but it IS something that has to happen whenever updating an open source package version.

And I believe that there should be no code changes associated with this change (I hope there are no code changes associated with this change).

Reply:
sounds good , will do